### ISSUE TITLE
Is it support PDF/A-3

### ISSUE THREAD

**Issue by bambootb** [tecnickcom/tc-lib-pdf#73]
Hi, would like to know does TCPDF support PDF/A-3. Or is there any way to convert your PDF to PDF/A-3 please advice thank you.

**Comment by mean-cj** [comment]
I agree. This update will add support for PDF/A-3 and Digital Signing.
